pygmt/enums.py

```diff
@@ -37,3 +37,21 @@ class GridFormat(IntEnum):
     GD = 22  #: Import through GDAL
     EI = 23  #: ESRI Arc/Info ASCII Grid Interchange format (ASCII integer)
     EF = 24  #: ESRI Arc/Info ASCII Grid Interchange format (ASCII float)
+
+
+class GridReg(IntEnum):
```

Comment:
`GridReg` or `GridRegistration`?

Reply:
I'd prefer `GridRegistration`

Reply:
OK, renamed to `GridRegistration` in this PR.